src/Controls/src/Build.Tasks/Controls.Build.Tasks-net6.csproj

```diff
@@ -33,13 +34,6 @@
 		<!-- <PackageReference Include="StrongNamer" Version="0.2.5" PrivateAssets="all" GeneratePathProperty="true" /> -->
 	</ItemGroup>
 
-	<PropertyGroup>
-		<PackageId>Microsoft.Maui.Controls.Build.Tasks</PackageId>
-		<IncludeBuildOutput>False</IncludeBuildOutput>
-		<NoWarn>$(NoWarn);NU5100;NU5128;</NoWarn>
```

Comment:
why we are removing this?

Reply:
This package no longer produces a NuGet package (uses `IsPackable=false`), so we can remove all NuGet-related properties.